### PR TITLE
Add Jenkinsfile and test file

### PR DESCRIPTION
**Pull request by MancunianSam**
This needs to be in its own repository because all of our other repositories have a build check which runs git secrets so if this file was in there, all future build checks would fail.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,12 @@
 # Git secrets check
+
+This repository is a Jenkinsfile which periodically runs git secrets against a test file. The test file contains 3 fake secrets.
+
+* A fake AWS account number
```
Comment (suzannehamilton) [inline README.md]: Should this be access key rather than account number?

File: Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  }
+
+  stages {
+    stage("Run git secrets") {
+      steps {
+        script {
+          def exitCode = sh(script: "git-secrets --scan", returnStatus: true)
+          def scanOutput = sh(script: "git-secrets --scan 2>&1 >/dev/null | head -n 4 | cut -c1-6 | tr -d '\n'", returnStdout: true)
+          if (exitCode == 0 || scanOutput.trim() != "test:1test:3test:5") {
+            tdr.postToDaTdrSlackChannel(colour: "danger", message: "*Security test failure*: expected Jenkins to detect sensitive information but it did not. Check that git-secrets is installed and configured on Jenkins.\nhttps://jenkins.tdr-management.nationalarchives.gov.uk/job/Git%20Secrets%20Check/")
```
Comment (suzannehamilton) [inline Jenkinsfile]: You can add a link to the actual build with the environment variable `BUILD_URL`.

**Review by suzannehamilton — COMMENTED** [review]
Just a couple of small comments, thanks.

File: README.md (hunk 1 above)
Comment (MancunianSam) [reply README.md]: Yep good spot, cheers.

File: Jenkinsfile (hunk 2 above)
Comment (MancunianSam) [reply Jenkinsfile]: Good point. I've changed it.

Comment (suzannehamilton) [reply Jenkinsfile]: Have you just updated the one below? I think it makes sense to use the build URL here too.

Comment (MancunianSam) [reply Jenkinsfile]: Ah yeah, sorry. I've updated the other message as well.

**Review by suzannehamilton — APPROVED** [review]

**Pull request merged**